docker-images/kafka/kafka-thirdparty-libs/2.1.x/pom.xml

```diff
@@ -52,16 +58,50 @@
             <groupId>io.opentracing.contrib</groupId>
             <artifactId>opentracing-kafka-client</artifactId>
             <version>0.1.1</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.apache.kafka</groupId>
+                    <artifactId>kafka-clients</artifactId>
+                </exclusion>
+            </exclusions>
         </dependency>
         <dependency>
             <groupId>io.strimzi</groupId>
             <artifactId>kafka-oauth-server</artifactId>
             <version>${strimzi-oauth-callback.version}</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.apache.kafka</groupId>
+                    <artifactId>kafka-clients</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>com.fasterxml.jackson.core</groupId>
+                    <artifactId>jackson-databind</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>com.github.spotbugs</groupId>
+                    <artifactId>spotbugs-annotations</artifactId>
```

Comment:
Is this not `provided` scope in `strimzi-kafka-oauth`? I think spotbugs should work if it is, but it won't become a transitive dependency since it's OK for annotations to be missing at runtime.

Reply:
It si not provided there. We also don't use it as provided in Strimzi AFAIK.

Reply:
I added strimzi/strimzi-kafka-oauth#10 to make it provided. In the mean time, we should keep this here since that change will be done only from 0.2.0 on and not for 0.1.0.